fighthealthinsurance/clinicaltrials_tools.py

```
@@ -0,0 +1,403 @@
+                nct_ids_json = json.dumps(nct_ids).replace("\x00", "")
+                await ClinicalTrialQueryData.objects.acreate(
+                    query=normalized_query,
+                    condition=norm_condition,
+                    intervention=norm_intervention,
+                    nct_ids=nct_ids_json,
+                )
+        except asyncio.TimeoutError:
+            logger.debug("Timeout in find_trials_for_query")
+        except asyncio.exceptions.CancelledError:
+            logger.debug("Cancelled in find_trials_for_query")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)